noether_msgs/CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ add_action_files(
   FILES
     GenerateToolPaths.action
     Segment.action
+    ThickSimulator.action
```

Comment:
We have been trying to keep action messages as a phrase about what they will do. eg SimulateThickness as opposed to thicknessSimulator